### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by tancor** [Driftt/drift-sdk-ios#43]
Hi,
my app crashes sometimes at ObjectValidator.swift with EXC_BREAKPOINT when accessing to pointer.pointee after deallocating the pointer.
```
fileprivate func dummyObject<T>() -> T{
        let pointer = UnsafeMutablePointer<T>.allocate(capacity: 0)
        pointer.deallocate(capacity: 0)
        return pointer.pointee
    }
```

**Comment by brianmcdon** [comment]
Thanks @tancor. Are you seeing this taking a particular action or on app launch.

**Comment by tancor** [comment]
Thank you for the response.
This happens on app launch.

**Comment by brianmcdon** [comment]
@tancor please try the newly released version 1.2.7
